query-engine/prisma/src/request_handlers/graphql/protocol_adapter.rs

```diff
@@ -145,7 +145,7 @@ impl GraphQLProtocolAdapter {
                     i
                 ))),
             },
-            Value::Float(f) => match Decimal::from_f64(f) {
+            Value::Float(f) => match Decimal::from_str(&f.to_string()).ok() {
```

Comment:
Discussed earlier, that this is a bug in `rust-decimal`. Maybe write a comment here?

Reply:
done